### DISCUSSION TITLE
Touch panel graph manipulation

### DISCUSSION THREAD

**Discussion by imotep33** [grafana/grafana#44353] [Dashboards, Panels & Visualizations]
Hi,
I would like to suggest a way to navigate into the graph with a touch device.
use 2 fingers to change the time range: zoom in, zoom out, and/or move left/right.
thanks.

**Comment by naymore** [comment]
In general I would love to see some mobile device optimizations.
e.g. why can I scroll to the right when there is only black screen space?

But **touch controls** to zoom in and out of a graph (change time frame and windowPeriod) would be most desired feature of all.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). I believe this is being discussed here, along with other mobile features https://github.com/grafana/grafana/issues/8799
